tests/kitchensink/bytes.proto

```diff
@@ -6,22 +6,23 @@ import "validate/validate.proto";
 message Bytes {
     bytes none = 1;
 
-    bytes min     = 2 [(validate.rules).bytes.min_len = 3];
-    bytes max     = 3 [(validate.rules).bytes.max_len = 10];
-    bytes min_max = 4 [(validate.rules).bytes = { min_len: 3, max_len: 10 }];
-    bytes eq_len  = 5 [(validate.rules).bytes = { min_len: 5, max_len: 5}];
+	bytes len     = 2 [(validate.rules).bytes.len = 5];
+    bytes min     = 3 [(validate.rules).bytes.min_len = 3];
+    bytes max     = 4 [(validate.rules).bytes.max_len = 10];
+    bytes min_max = 5 [(validate.rules).bytes = { min_len: 3, max_len: 10 }];
+    bytes eq_len  = 6 [(validate.rules).bytes = { min_len: 5, max_len: 5}];
 
-    bytes pattern  = 6 [(validate.rules).bytes.pattern  = "foo+(ba+r)?"];
```

Comment:
Renumbering proto fields is a no-go since it breaks compatibility. It probably wouldn't matter for most consumers of this library since it's used to annotate source code, but it'd be much better to leave the existing fields as they are and add `len` with the next available field number (16, if I read this right). Don't worry about the field numbers being out of order.

Reply:
So I will just make `len` as the number 16 not consider order

Reply:
Exactly.